docs/sources/installation/upgrading.md

```diff
@@ -189,8 +189,8 @@ this new setting.
 ### Session storage is no longer used
 
 In 6.2 we completely removed the backend session storage since we replaced the previous login session implementation with an auth token.
-If you are using Auth proxy with LDAP an shared cached is used in Grafana so you might want configure [remote_cache] instead. If not
-Grafana will fallback to using the database as an shared cache.
+If you are using Auth proxy with LDAP, a shared cached is used in Grafana, so you might want to configure [remote_cache] instead. If not,
+Grafana will fallback to using the database as a shared cache.
```

Comment:
Use "fall back"

Reply:
Agreed. "Fall back" is a verb, "fallback" is a noun. It's similar to "set up" and "setup."

Reply:
Can you change "fallback" to "fall back"? Thank you.

Reply:
I have made the changes. Thanks for the suggestion @achatterjee-grafana